libplatsupport/src/plat/bcm2711/clock.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017, Data61, CSIRO (ABN 41 687 119 230)
+ * Copyright (C) 2021, Hensoldt Cyber GmbH
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#include <platsupport/i2c.h>
+
+#include "../../arch/arm/clock.h"
+#include "../../services.h"
+#include <assert.h>
+#include <string.h>
+#include <utils/util.h>
+
+static struct clock master_clk = { CLK_OPS_DEFAULT(MASTER) };
+static struct clock sp804_clk = { CLK_OPS_DEFAULT(SP804) };
+
+int
+clock_sys_init(ps_io_ops_t* o, clock_sys_t* clock_sys)
+{
+    clock_sys->priv = (void*)0xdeadbeef;
+    clock_sys->get_clock = &ps_get_clock;
+    clock_sys->gate_enable = NULL;
+    return 0;
+}
+
+void
+clk_print_clock_tree(clock_sys_t* sys)
+{
+    clk_t *clk = clk_get_clock(sys, CLK_MASTER);
+    clk_print_tree(clk, "");
+}
+
+clk_t* ps_clocks[] = {
+    [CLK_MASTER]   = &master_clk,
+    [CLK_SP804]    = &sp804_clk,
+};
+
+/* These frequencies are NOT the recommended frequencies. They are to be used 
+ * when we need to make assumptions about what u-boot has left us with. 
+ * These values originate from the bcm2837. According to the BCM2837 TRM in 
```

Comment:
better say "from the bcm2837 platform port"

Reply:
Fixed: https://github.com/seL4/util_libs/pull/60/files#diff-80a1b0673b7adaebd6fc7e47ed20325d3052a0b3212ee7a3c012288a6d030c36R41